### ISSUE TITLE
Alcoholic beverages in FOODON 

### ISSUE THREAD

**Issue by JessSing** [FoodOntology/joint-food-ontology-wg#17]
I noticed that there are three different classes in FOODON for beer: 'beer beverage' (00001260), 'beer' (3301619), and 'beer product' (00001064). 

I found some inconsistencies with 'beer' (#3301619) since 'beer' is next to 'beer draft' and 'beer light' which doesn't make sense since the latter two are types of the former.

I subclassed the beers in the Beer Ontology under 'beer beverage' (00001260) and 'fermented plant beverage' (00001254).

While they aren't technically beers I have also added cider and subclassed it under 'fermented plant beverage' (00001254) as well as mead, which is subclassed under fermented beverage (00001094) since it is made by fermenting honey (not a plant).

Cross references ticket Myra-Analytics/beer#1 in the Beer Ontology

**Comment by cmrn-rhi** [comment]
> I noticed that there are three different classes in FOODON for beer: 'beer beverage' (00001260), 'beer' (3301619), and 'beer product' (00001064).

> I subclassed the beers in the Beer Ontology under 'beer beverage' (00001260) and 'fermented plant beverage' (00001254).

I agree with you with focusing on `beer beverage` (00001260) and it's parent class `fermented grain beverage` (00001102) of the `fermented plant beverage` (00001254) class. The `beer` (3301619) was an imported term that had yet to be reviewed; I think it can be integrated into `beer beverage` (00001260), plus the parent class `barley malt beverage` (00002017) isn't appropriate as not all beers are barley food products.

I'll reconfigure these in FoodOn within the next day or two.

> I found some inconsistencies with 'beer' (#3301619) since 'beer' is next to 'beer draft' and 'beer light' which doesn't make sense since the latter two are types of the former.

I concur, these are more suitable as subclasses.

> While they aren't technically beers I have also added cider and subclassed it under 'fermented plant beverage' (00001254) as well as mead, which is subclassed under fermented beverage (00001094) since it is made by fermenting honey (not a plant).

I agree with the placement of `mead` under `fermented beverage` (00001094), this is also where `mead wine` (03307306) is situated. I think that `mead wine` (03307306) would likely be a subclass of `mead` and possibly `wine`... but that leads down the rabbit hole of whether mead is honey wine, and if not where those blurry boundaries are. I'll end that segue here for now but make a note on returning to this after the beer discussion is resolved.

As for `cider`, FoodOn does already have `apple cider (alcoholic)` (00001276) nested within `fermented plant beverage` (00001254).

**Comment by cmrn-rhi** [comment]
@JessSing 

Sorry for the delay. Many of the aforementioned changes have been implemented, with `beer beverage` (00001260) becoming the primary beer class to which the other beers are now subclasses of, and `beer` (3301619) being changed to be `barley malt beer beverage` (3301619). More details available at [JFO #17 Resolution FoodOntology/foodon#125](JFO #17 Resolution FoodOntology/foodon#125).

Some additional information was added to `porter` (03315627), and a few more beer related terms (`lager`, `stout`, and hopefully `mead`) will be added in the near future.

**Comment by mateolan** [comment]
Not sure this issue should be closed. It seems like the beer category should be built out according to, or at least mapped to, the various official and semi-official style guidelines. 

Here is a partial list:

Brewers Association: https://cdn.brewersassociation.org/wp-content/uploads/2020/03/BA-beer-style-guidelines-2020.pdf

Japan: http://beertaster.org/beerstyle/web1804/beerstyle_main1804_e.html

AIBA: Note the Australian InternationalBeer Awards (AIBA) is the largest annual competition in the world: https://web.archive.org/web/20180323174844/http://rasv.com.au/Events/AIBA_Home/AIBA_Entries/AIBA_Publications/AIBA_2018_EntryBooklet

Note that the BJCP (Beer Judging Certificate Program) also has mead and cider styles with ingredient reference. There are spreadsheets here too, so the import should be straightforward. https://www.bjcp.org/stylecenter.php

Craftbeer.com offers a nice style guide (https://www.craftbeer.com/beer/beer-styles-guide)  as well as some beautiful pics (https://www.craftbeer.com/beer-styles) these line-up nicely (but not perfectly) with brewersassociation.org and bjcp .org references

Best,

~Matthew

**Comment by cmrn-rhi** [comment]
@mateolan I agree that we should leave this issue open. The beer branch definitely needs more fleshing out and the resources you provided will be very helpful, thank you!

**Comment by JessSing** [comment]
@cmrn-rhi thanks for changes. I agree with everything above, except I don't think 'mead' should be a subclass of 'Beer'. It should be a food product since it is fermented honey, which isn't a plant.

**Comment by mateolan** [comment]
@JessSing totally right. Additionally, .the fermentation process and products needs rethinking from a microbiologists perspective. That is, these products should be classed in terms of the types of fermentations. For instance:
fermentation condition: e.g. aerobic/anaerobic
fermenting organism: e.g. type of yeast/bacteria/mold

Depending on the ferm type/organism/etc, the fermentation will yield acid, alcohol, or a variety of other products...and have significant impact on chemical qualities and sensory attributes

**Comment by ddooley** [comment]
I think mead is actually FoodOn's = "mead wine", since on the Net there seems to be no other sense of "mead wine" that doesn't point to mead?

**Comment by cmrn-rhi** [comment]
> @cmrn-rhi thanks for changes. I agree with everything above, except I don't think 'mead' should be a subclass of 'Beer'. It should be a food product since it is fermented honey, which isn't a plant.

I agree it shouldn't be under beer, my suggestion was placing it where `mead wine` (FOODON:03307306) is under `fermented beverage` (FOODON:00001094). I also agree that we shouldn't but it under `plant derived fermented beverage` (FOODON:00001254) since honey is an animal food product.
**EDIT:** I've just noticed that `mead wine` (FOODON:03307306) appears directly under `fermented beverage` (FOODON:00001094) when viewing in Protégé but not in EBI. In EBI it is nested under `wine or wine-like food product` (FOODON:00002364) > `non-fruit wine food product` (FOODON:00001259); note, it is also subclassed here in Protégé.

The only reason I held off on adding it immediately is because there appears to be a debate within the mead community whether `mead` = `mead wine` = `honey wine1 (@ddooley). Some consider it a completely separate beverage from wine and I wanted to take a little more time to dig into it. I think it mostly stems from people having a strong attachment to something only qualifying as a wine if it is derived from fruit, or some definition only with grapes (but there are many fruit wines that make this unreasonable; e.g. apple wine, rice wine, etc.).

In FoodOn `wine or wine-like food product` (FOODON:00002364) is not so constrained as those definitions, but is considered a `fruit based alcoholic beverage` (FOODON:00001882) [and thus a `plant derived fermented beverage` (FOODON:00001254)] which @JessSing and I agree `mead` is not (unless it is a mead that also contains fruit - in my opinion). 

It also had me thinking about at what point does `cider` cross the line into `apple wine` (this [source](https://www.myrecipes.com/extracrispy/whats-the-difference-between-apple-cider-and-apple-wine#:~:text=Apple%20wine%20isn't%20radically,process%2C%20but%20requires%20more%20sugar.&text=Apple%20cider%20has%20pulp%20and,be%20the%20first%20to%20know.) considers the alcohol content) and whether we can use a similar defining line here.

Would love to hear everyone's thoughts.

**EDIT:** There is also `wine (honey)` (FOODON:03315687) that can be integrated into the appropriate term once this `mead` discussion is resolved.

**Comment by cmrn-rhi** [comment]
> I think mead is actually FoodOn's = "mead wine", since on the Net there seems to be no other sense of "mead wine" that doesn't point to mead?

I can follow up with sources later today when I have some time.

**Comment by maweber-bia** [comment]
I will try to get some information about cider vs apple wine since in the lab we have a team working closely with the technical Center for cider
[see here](https://www6.angers-nantes.inrae.fr/bia_eng/BIA-highlights/Partnership/UMT-Nova2cidre)

Please also notice that in France, the use of the word "wine" is very strict, and should be reserved for the alcoholic drink typically made from fermented grape juice. 
[see here](https://www.oenologie.fr/droit/legislation/definition-vin.php)

**Comment by maweber-bia** [comment]
As for mead,  in French, we have "hydromel" or "chouchen", which are **fermented honey drinks**.
I think that in English "mead", "mead wine" and "honey wine" could/should be equivalent (synonyms), and  "fermented beverage" (FOODON:00001094) the only correct parent class...but that is my own opinion

**Comment by oldskeptic** [comment]
In FoodOn is a synonym an additional term linked to the primary term or an additional label?

**Comment by ddooley** [comment]
Synonyms are additional terms linked to a primary term using the oboInOwl:hasSynonym, oboInOwl:hasExactSynonym, oboInOwl:hasNarrowSynonym (for terms we haven't necessarily gotten around to curating, or don't have enough information on, or or brand things out of our purview), and oboInOwl:hasBroadSynonym.  Both hasSynonym and hasBroadSynonym are being discouraged/phased out in favour of the other two.

**Comment by ddooley** [comment]
We anticipate additional labels will only exist for language translations of the primary (english) label.

**Comment by cmrn-rhi** [comment]
### Re: Mead

*Sorry for the delay in my following-up. Here are some sources, with "formal" being technical, a publication, or from someone that appear to be an expert; while "informal" is not necessarily any of the former*

<details>
<summary>
<b>Sources that consider mead it’s own category: </b>
</summary>
<p>

- (formal) [Commercial Mead Production - Wines Vines Analytics](https://winesvinesanalytics.com/features/article/110520/Commercial-Mead-Production)
  - “People are often confused by mead: Is it wine, or is it beer? Historically, it has been known as both. While the primary ingredients of mead are honey and water, the basic recipe has been augmented with an array of other ingredients, many of which are common to either wine or beer, such as fruit, hops and barley. In addition, some production methods employ boiling, an activity more akin to beer brewing than winemaking. In reality, mead blurs the lines between wine and beer and makes it difficult to assign it to one realm or the other.”
- (formal) [Developments in the Fermentation Process and Quality Improvement Strategies for Mead Production](https://www.mdpi.com/1420-3049/19/8/12577/htm)
  - Treats mead like a separate entity, even stating “Fermented drinks obtained from honey, among which mead, are thought to be the oldest alcoholic beverages known to man, made thousands of years before either wine or beer were produced.”
- (informal) https://www.liquor.com/articles/10-facts-about-mead/
  - Consider all meads (even one with additional fruits) not to be wines.
- (informal) [Mead vs Wine: What is the Difference?](https://tanglewoodwine.co.uk/blogs/news/mead-vs-wine-what-difference#:~:text=Wine%20is%20made%20from%20fermented,other%20ingredients%20alongside%20the%20honey.)
  - Based on the definition of wine being from fermented grapes.
- (informal) [It’s Not “Honey Wine” or “Honey Beer”… So, What’s Mead?](https://www.pollenangels.com/blog/if-its-not-honey-wine-or-beer-whats-mead)
  - Says “The agreed-upon definition at this time is that mead is a fermented beverage made with honey, much like wine is made from grapes, cider from apples, beer from grains, sake from rice, etc. This makes mead similar to wine or beer at times, such as when you drink a pyment mead made with grapes or grape juice or a braggot made with hops, but mead is still its own type of beverage – after all, beer isn’t a grain wine, nor is wine a grape beer!” but then goes on to reference the AMMA which does consider it a wine…  

</p>
</details>

<details>
<summary>
<b>Otherwise, there are lots of formal and informal sources that consider mead a honey wine.</b> Here are just a couple (US centric):
</summary>
<p>

- (formal) [American Mead Makers Association](https://mead-makers.org/about-mead/?v=3e8d115eb4b3#:~:text=Simply%20put%2C%20mead%20is%20a,wine%20by%20THOUSANDS%20of%20years.) - What Exactly is Mead?
  - “Mead is a wine from other agricultural products or a malt beverage derived (1) from honey and water, OR (2) from a mixture of honey and water with hops, fruit, spices, grain, or other agricultural products or flavors allowed in the production in wine, in which honey represents the largest percentage of the starting fermentable sugars by weight of the finished product, and having the taste, aroma, and characteristics generally attributed to mead, and sold or offered for sale as mead.”
- (formal) [US Alcohol and Tobacco Tax and Trade Bureau](https://www.ttb.gov/faqs/alcohol#hw) (and the FAA, FDA)
  - ‘For labeling purposes, it is TTB's policy to allow either the term "honey wine" or "mead" as the designation for honey wine that complies with the standards of identity for honey wine set forth in 27 CFR 4.21(f). Furthermore, the terms may be used interchangeably as part of a statement of composition for a wine specialty product (for example, "honey wine with natural cherry flavor" or "mead with natural cherry flavor").’

</p>
</details>

---

>  think that in English "mead", "mead wine" and "honey wine" could/should be equivalent (synonyms), and "fermented beverage" (FOODON:00001094) the only correct parent class...but that is my own opinion

I agree with @maweber-bia that we should treat `mead wine` and `honey wine` as synonyms for `mead`, since historically (and presently), they are so interwoven. I also agree with having `fermented beverage` (FOODON:00001094) as the parent class; mostly because I think `mead` is it's own thing to a similar degree of separation that exists between `beer` and `wine` which we treat as more separate entities. But that's also just my opinion! 

**UPDATE 2021-29-01** Have a slight shift in this opinion, as I realize I need to take a deeper dive into some of the other resources listed here, but I think having `mead` under `non-fruit wine food product` (FOODON:00001259) - which is in-progress of being moved under `fermented beverage` (FOODON:00001094) directly and out of `fruit based alcoholic beverage` (FOODON:00001882) - would be appropriate.

**Comment by cmrn-rhi** [comment]
### Re: Wine

> Please also notice that in France, the use of the word "wine" is very strict, and should be reserved for the alcoholic drink typically made from fermented grape juice.

@maweber-bia do you know if France has a classification for the fermented beverages that other countries might consider "none-grape wines"?

**Comment by maweber-bia** [comment]
> 
> ### Re: Wine
> 
> > Please also notice that in France, the use of the word "wine" is very strict, and should be reserved for the alcoholic drink typically made from fermented grape juice.
> 
> @maweber-bia do you know if France has a classification for the fermented beverages that other countries might consider "none-grape wines"?

I will try to find it but for sure, we have clear distinctions in the "alcoholic" and non-alcoholic" beverage sectors as regards the legislative rules (for example, for non-alcoholic beverages, what a "juice" is, or a "nectar", etc.)

**Comment by oldskeptic** [comment]
@cmrn-rhi I confirm that the UK [CAMRA](https://camra.org.uk/) still uses the word mead. 

@maweber-bia A set of terms that define things according to the different legislations would be useful in aligning / differentiating food across jurisdictions.

**Comment by maweber-bia** [comment]
> > ### Re: Wine
> > > Please also notice that in France, the use of the word "wine" is very strict, and should be reserved for the alcoholic drink typically made from fermented grape juice.
> > 
> > 
> > @maweber-bia do you know if France has a classification for the fermented beverages that other countries might consider "none-grape wines"?


**Hereafter are the comments from the INRAE expert  about the denomination "Apple wine" :**

It was mainly the Germans who used Apfel Wein a lot. Apfel Wein is easily translated as Apple wine but "cider" is the classic term for fermented apple products. From a cultural point of view, the terms cidre (French), cider (English), chistr (Breton) and other related terms derive from a Hebrew term meaning fermented product.  It should also be pointed out that the Americans have caused confusion by calling their fresh apple juice "cider". So it's a mess! And I'm not convinced that there is a difference for the English between cider and apple wine: logically all should be fermented drinks obtained by the alcoholic fermentation of apple juice more or less diluted and sweetened (it is estimated that in English cider there is only 30% apple juice, the rest being added sugar). French cider is obtained only from apple juice (pure juice cider) but for ciders without this indication the apple juice can be diluted and/or apple juice concentrate added up to a limit of 50% of the initial sugars. 
(Translated with www.DeepL.com/Translator - free version)

**For apple-based alcoholic drinks**, according to the [French legislation](https://www.legifrance.gouv.fr/loda/id/LEGIARTI000006557454/1992-08-22/#LEGIARTI000006557454), we have the following definitions:

**[article 3**](https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000006557454/1992-08-22)
Flavoured apple-based alcoholic drinks are considered to be
1° With an actual alcoholic strength by volume of more than 1% and not more than 7%.  ;
2° Have a total alcoholic strength by volume of not more than 11.5%;
3° Obtained from apple, apple must, or a mixture of fresh apple and pear must, concentrated apple must, or a mixture of concentrated apple or pear must, apple fermented or cider in a minimum proportion of 50% of the finished product;
4° Flavoured from fruit or vegetable juices, natural flavouring substances, flavouring substances identical to natural ones, or flavouring preparations mentioned in Decree No. 91-366 of 11 April 1991 or their mixture. The product obtained must have organoleptic characteristics different from those of the products mentioned in 3°;
5° With a sugar content expressed in saccharose of not more than 80 grams per litre.

Sugars, honey and water may be used for the manufacture of the beverages defined in this article.
(Translated with www.DeepL.com/Translator - free version)

_Please also note that the fermented beverage made from pears is called "Poiré" in France._

Besides, at the EU level, I've found a regulation on [**aromatised wine products**](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32014R0251&rid=1#d1e408-14-1)

Another one for [**spirit drinks** :](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32019R0787&rid=2#d1e541-1-1)

I am searching for their "translation" into the French jurisdiction. Coming soon, hopefully!

**Comment by cmrn-rhi** [comment]
# General Update

## What's been done so far
* `beer beverage` (FOODON:00001260) has been made the dominate `beer` term with all other related beers as subclasses.
* `alcoholic beverage` (FOODON:00001579) has been moved out from under `plant derived fermented beverage` (FOODON:00001254) into `fermented beverage` (FOODON:00001094). This was done because not all alcoholic beverages are plant derived (e.g. honey or dairy fermented drinks above 0.5% alcohol).

## New Alcoholic Beverages Overview

The overarching alcoholic beverages categories will be kept broad, being define along the lines of whether they have *wort*, *must*, or *distillation* as a key component in their creation. Alcohol ranges will also be broad to be inclusive of different agency definitions. (labels have been simplified)

* `Alcoholic Beverage` (FOODON:00001579)
  * `Beer` (FOODON:00001260)     <-- **wort**
  * `Wine` (FOODON:00002364)    <-- **must**
  * `Spirits` (FOODON:03315585)  <-- **distilled**
  
This branch will be further broken down as follows:
* `Alcoholic Beverage` (FOODON:00001579)
  * `Beer` (FOODON:00001260)
  * `Wine` (FOODON:00002364)
    * `Fruit Wine` (FOODON:03315334)
      * `Grape Wine` (FOODON:00001925)
      * `Apple Wine/Cider` (FOODON:03301134)
    * `Non-Fruit Wine` (FOODON:00001259)
      * `Honey Wine/Mead` (FOODON:03307306)
      * `Rice Wine` (FOODON:03307312)
  * `Spirits` (FOODON:03315585)

Additional subclasses will be added eventually as well.

Curation of a `Non-Alcoholic Beverage` branch will also be done to include things like `mixers` and `alcoholic beverage analog` (FOODON:00001580)

## Agency Mappings

After the branch is restructured, we'll begin adding additional alcohol related agency mappings (using resources from this discussion thread to begin with) from across the globe.

Example:
* `Agency Food Product type` (FOODON:03400361)
  * `Brewers Association Beer Style Guidelines`
  * `Craft Beer Beer Style Guidelines`
  * `etc…`
  * `European Union Agency Food Product Type` (FOODON:03400356)
    * `EEC Alcohol Regulations`
    * ...

This will be done transparently and in collaboration with agency experts.

## Summary Image
![image](https://user-images.githubusercontent.com/48695054/106795783-62057700-660f-11eb-865e-c7979bf8593b.png)

(From **FoodOn Curation Meeting Presentation**. Slides are available for viewing [**here**](https://docs.google.com/presentation/d/1y3S6xklV283JCMsi3dmjhQwF2kFlo_Oz3mbkpbgmwd4/edit?usp=sharing))

**Comment by cmrn-rhi** [comment]
### RE: Fermentation

> Additionally, .the fermentation process and products needs rethinking from a microbiologists perspective. That is, these products should be classed in terms of the types of fermentations. For instance:
> fermentation condition: e.g. aerobic/anaerobic
> fermenting organism: e.g. type of yeast/bacteria/mold
> 
> Depending on the ferm type/organism/etc, the fermentation will yield acid, alcohol, or a variety of other products...and have significant impact on chemical qualities and sensory attributes

Definitely more to be discussed on this topic as well. However, since this topic also applies to non-alcoholic beverages/food products (and because I'm concerned this issue thread might get unwieldy) would it be alright with everyone if I moved this to a new issue request?

**Comment by ddooley** [comment]
Fine by me.  That's the usual practice - to close primary issue if resolved, and add other threads for related issues.

**Comment by mateolan** [comment]
I like where you are going...But many questions remain:
What percent alcohol needs to be in the food for you to consider it an
alcoholic beverage?
Also, aside from must and wort, are there other distinctions between beer
and wine (usually it is ? With rice wine and barley wine both validated
products--but is rice wine made with a must? The lines are being blurred.
<https://www.sfchronicle.com/wine/article/New-California-law-makes-it-easier-to-blur-the-14931607.php>
Where are you putting Kombucha and Kumis (both of which **may be** alcoholic
and neither use must nor wort --Kombucha uses a SCOBY [acronym for
symbiotic culture of bacteria and yeast])? I am sure there are other
examples.
Also what about "fortified" wines and beers that have distilled alcohol
added back into them?
..and then of course there are cocktails...

**Comment by cmrn-rhi** [comment]
> What percent alcohol needs to be in the food for you to consider it an alcoholic beverage?

Currently going with > or >= 0.5% as that seems to be what the majority of countries consider alcoholic. Since we are trying to keep the major classes broad I am going with alcoholic being >= 0.5% and non-alcoholic being < 0.5. Will adjust if necessary. 

> Also, aside from must and wort, are there other distinctions between beer and wine (usually it is ?

As you say, the lines are very blurry. I personally cannot find a line that isn't blurred somehow somewhere (historically, culturally, geographically, in composition, opinion etc.) - the best line I've found is in the brewing process.

I'm using `must` in the broadest sense, that is the prefermentation product that has been processed into a juice-like state prior to fermentation. While some may limit the definition to grapes, or fruit (as people do with wine), this is not inclusive enough and it can be structured similarily to how `wine` > `fruit wine` > `grape wine` is. *Examples of must in FoodOn: `grape must` (FOODON:00001071), `apple must` (FOODON:00002391), `oat must` (FOODON:03317168)...

However, to simplify it further one can simple consider `wine` is anything alcoholic derived directly from the fermentation of sugars that aren't derived from `wort` (which was originally my thought process).`wort` is similar to `must` (broad version) except for it begins with a malting process. `beer` also involves the addition of aromatics. `hops` have been used (at minimum for aromatic content) for centuries and some only consider `beer` to be `beer` if it contains `hops` (otherwise some other source of aromatics/bitterness where it's called `gruit`) - but there are edge cases (e.g. `all-barley beer`).

Going with alcohol % isn't great as there is too much overlap in the ranges between `beer`, `wine`, and `spirits`. Alcohol ranges for the broadest will be determined after agency materials are review. In my initial pass brewing and age times also not give a solid line to work with between `beer` and `wine` but that may prove otherwise as investigations go further.

> but is rice wine made with a must?

I have read brewers who call it `must`, but again it depends on how one defines `must`. If one disagrees with this definition, it can still work with the "isn't made from `wort`" definition as the process of making rice wine does not involve malting (as the components of the rice kernel required for malting are removed).

> Where are you putting Kombucha and Kumis (both of which \*\*may be\*\* alcoholic and neither use must nor wort --Kombucha uses a SCOBY \[acronym for symbiotic culture of bacteria and yeast\])?

Kombucha tea and Kumis are designated as non-alcoholic when <0.5%. Otherwise `Kumis` and `Alcoholic Kombucha Tea`would go under `non-fruit wine` under the "not from wort" definition and that is ferments directly from sugars.

All of these alcoholic beverages contain yeast, but as the fermentation discussion develops we can consider using additional microorganisms and fermentation processes for classification lines. Further study may reveal  `Alcoholic Kombucha Tea` should be it's own class because the alcohol content is derived from the second fermentation, not the first (based on my knowledge so far).

>  Also what about "fortified" wines and beers that have distilled alcohol added back into them? ..and then of course there are cocktails...

Those will definitely have to be discussed and may end up with multiple parents or a hybrid parent class (even beer/wine hybrids are a thing now). `cocktails` and `mocktails` will find a homes as well, likely under some sort of `mixed beverage` category. It is yet to be determined.

The things list in the overview are the starting point. Next stages are reviewing all terms within said branch and reposition, reviewing agency information (to inform agency terms and the general class ranges), and continuing to discuss oddities.

This isn't going to be done in one go. I'm going to start redeveloping the branch (in isolation of the rest of FoodOn) and as it gets developed I'll return to this thread with progress updates and discussion of outliers. Revisions will occur throughout this process. I will also share a spreadsheet with all the terms being worked on once I've started, that way everyone can look for outliers/revisions on their own as well.

**Comment by cmrn-rhi** [comment]
> Fine by me. That's the usual practice - to close primary issue if resolved, and add other threads for related issues.

I'd like to keep this issue open to continue alcoholic beverage discussions for now so there's a place to discuss outliers/items of concern. Or would it be better to open new issues as they arise?

That aside I'll open a new fermentation issue. We could still discuss fermentation concerns for alcohol here, but I think to general brainstorm/development should happen in another thread.

**Comment by ddooley** [comment]
Whichever way you feel most comfortable about in managing the info!

**Comment by maweber-bia** [comment]
> 
> > but is rice wine made with a must?
> 
> I have read brewers who call it `must`, but again it depends on how one defines `must`. If one disagrees with this definition, it can still work with the "isn't made from `wort`" definition as the process of making rice wine does not involve malting (as the components of the rice kernel required for malting are removed).


> > Where are you putting Kombucha and Kumis (both of which **may be** alcoholic and neither use must nor wort --Kombucha uses a SCOBY [acronym for symbiotic culture of bacteria and yeast])?


[From Wikipedia](https://en.wikipedia.org/wiki/Must), "Must (from the Latin vinum mustum, "young wine") is freshly crushed fruit juice (usually grape juice) that contains the skins, seeds, and stems of the fruit. (.../...)
This term is also used by meadmakers for the unfermented honey-water mixture that becomes mead. The analogous term in beer brewing is wort. "

"Must" could easily be applied to fruit wines and honey wine (which is a kind of "exception") but it seems that it could also be used for vinegar (which results from another acetic fermentation process). So, it is a bit confusing.

Therefore, I think the 'non-wort' definition would perhaps be more appropriate to include also any other  non-fruit fermented beverages and also those from animal origin like Kumis

> Kombucha tea and Kumis are designated as non-alcoholic when <0.5%. Otherwise `Kumis` and `Alcoholic Kombucha Tea`would go under `non-fruit wine` under the "not from wort" definition and that is ferments directly from sugars.
> 


By "ferments directly from sugars, I guess you mean there is no need for germination as in the malting process ?

**Comment by JessSing** [comment]
Thank you for this! FOODON and Beer now align on these terms and allow for cross-reference to beer, cider and mead. I think the current definitions are generic enough to build on and handle future corner cases like Root Beer, Spruce Beer (soda) and Spruce Beer (alcoholic). Different countries have national and local standards for what beer, wine (or champagne) should be, but perhaps these should be specialization classes?

@oldskeptic will publish the beer ontology this morning and references to FOODON.

**Comment by cmrn-rhi** [comment]
> Therefore, I think the 'non-wort' definition would perhaps be more appropriate to include also any other non-fruit fermented beverages and also those from animal origin like Kumis

Great, I'll move forward with this definition.

> By "ferments directly from sugars, I guess you mean there is no need for germination as in the malting process ?

Yes! :)

**Comment by cmrn-rhi** [comment]
> Different countries have national and local standards for what beer, wine (or champagne) should be, but perhaps these should be specialization classes?

That's what I'm hopping to address by mapping agency definitions to different branches, but specialization classes may be warranted down the line.
